src/test/kotlin/uk/gov/justice/digital/hmpps/incentivesapi/service/PrisonerIepLevelReviewServiceTest.kt

```diff
@@ -657,48 +658,62 @@ class PrisonerIepLevelReviewServiceTest {
       )
       whenever(prisonApiService.getPrisonerInfo("A1244AB", true))
         .thenReturn(prisonerAtLocation)
+
+      val newReview = PrisonerIepLevel(
+        id = 1L,
+        prisonerNumber = "A8765SS",
+        bookingId = 1234567L,
+        prisonId = "LEI",
+        locationId = "LEI-1-1-001",
+        reviewedBy = "TEST_STAFF1",
+        iepCode = "BAS",
+        current = true,
+        reviewTime = LocalDateTime.now().minusDays(2)
+      )
+      val oldReview1 = PrisonerIepLevel(
+        id = 3L,
+        prisonerNumber = "A1244AB",
+        bookingId = 555555L,
+        prisonId = "LEI",
+        locationId = "LEI-1-1-001",
+        reviewedBy = "TEST_STAFF1",
+        iepCode = "STD",
+        current = true,
+        reviewTime = LocalDateTime.now().minusDays(100)
+      )
+
+      val oldReview2 = PrisonerIepLevel(
+        id = 2L,
+        prisonerNumber = "A1244AB",
+        bookingId = 555555L,
+        prisonId = "LEI",
+        locationId = "LEI-1-1-001",
+        reviewedBy = "TEST_STAFF1",
+        iepCode = "BAS",
+        current = false,
+        reviewTime = LocalDateTime.now().minusDays(200),
+      )
+
       whenever(prisonerIepLevelRepository.findAllByPrisonerNumberOrderByReviewTimeDesc("A8765SS"))
         .thenReturn(
           flowOf(
-            PrisonerIepLevel(
-              prisonerNumber = "A8765SS",
-              bookingId = 1234567L,
-              prisonId = "LEI",
-              locationId = "LEI-1-1-001",
-              reviewedBy = "TEST_STAFF1",
-              iepCode = "BAS",
-              current = true,
-              reviewTime = LocalDateTime.now().minusDays(2)
-            )
+            newReview
           )
         )
+
       whenever(prisonerIepLevelRepository.findAllByPrisonerNumberOrderByReviewTimeDesc("A1244AB"))
         .thenReturn(
           flowOf(
-            PrisonerIepLevel(
-              prisonerNumber = "A1244AB",
-              bookingId = 555555L,
-              prisonId = "LEI",
-              locationId = "LEI-1-1-001",
-              reviewedBy = "TEST_STAFF1",
-              iepCode = "BAS",
-              current = false,
-              reviewTime = LocalDateTime.now().minusDays(200),
-            ),
-            PrisonerIepLevel(
-              prisonerNumber = "A1244AB",
-              bookingId = 555555L,
-              prisonId = "LEI",
-              locationId = "LEI-1-1-001",
-              reviewedBy = "TEST_STAFF1",
-              iepCode = "STD",
-              current = true,
-              reviewTime = LocalDateTime.now().minusDays(100)
-            )
+            oldReview2,
+            oldReview1
```

Comment:
I don't think it matters for this test but the order would be `flowOf(oldReview1, oldReview2)` because the query sort by review time desc and `oldReview1` is more recent (100 days old) than `oldReview2` (200 days old)

...but as I said, it shouldn't impact the test as the merge copy all of them reviews with the old `prisonerNumber` regardless :)